packages/rancher-alerting-drivers/charts/templates/cluster-role.yaml

```
@@ -0,0 +1,50 @@
+{{- if and (not .Values.sachet.enabled) (not .Values.prom2teams.enabled) -}}
+{{- fail "At least one Driver must be enabled to install the chart. " }}
+{{- end -}}
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ printf "%s-admin" (include "drivers.fullname" .) }}
```

nit: you could just change this to:
```suggestion
  name: {{ include "drivers.fullname" . }}-admin
```

---

fixed.